mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1961,16 +1970,6 @@ def evaluate(
                 error_code=INVALID_PARAMETER_VALUE,
             )
 
-        model_predict_func = None
-        if model is not None and hasattr(model, "predict") and callable(model.predict):
-
-            def model_predict_func(x):
-                # In non-langchain environments, nothing would be autologged.
-                with mlflow.utils.autologging_utils.restrict_langchain_autologging_to_traces_only():
-                    return model.predict(x)
-
-        self.model_predict_fn = model_predict_func
```

Comment:
Having this attribute with `self.predict_fn` and `self.predict_proba_fn` is confusing. Removed.